lib/ace/mode/css/csslint.js

```diff
@@ -1877,10 +1877,13 @@ Parser.prototype = function() {
 
                     value = new PropertyName(propertyName, hack, line || token.startLine, col || token.startCol);
                     this._readWhitespace();
-                } else if (tokenStream.peek() === Tokens.RBRACE) {
-                    // Encountered when there are no more properties.
                 } else {
-                    this._unexpectedToken(tokenStream.LT(1));
+                    var tt = tokenStream.peek();
+                    if (tt === Tokens.EOF || tt === Tokens.RBRACE) {
```

Comment:
it's a bit strange to have an empty if block, better inverse the condition to 
```
if (tt !== Tokens.EOF && tt !== Tokens.RBRACE)
```

Reply:
Yes I see that but I didn't wanted to change this odd syntax to preserve the comment logic. I just push the modification.